docs/api.rst

```diff
@@ -406,7 +406,25 @@ Array form of :ref:`zip`.  Apply a function to corresponding events from all the
   s2:                           --1--2--3->
   zipArray(add3, [s1, s2, s3]): --3--6--9->
 
-sample :: ((a, b) -> c) -> Stream a -> Stream b -> Stream c
+sample
+^^^^^^
+.. code-block:: haskell 
+
+  sample :: ((a, b) -> c) -> Stream a -> Stream b -> Stream c
+
+Create a new stream by combining sampled values from many input streams.::
```

Comment:
Sometimes we attempt to convey a lot of information in just one sentence, but perhaps splitting it up into a more natural language of more sentences will help.